js/phemex.js

```diff
@@ -1012,6 +1012,9 @@ module.exports = class phemex extends Exchange {
         };
         const duration = this.parseTimeframe (timeframe);
         const now = this.seconds ();
+        // Note !
+        // * this exchange doesnt support native API endpoint for kline data, so this implementation uses data from Web-UI chart, 
+        // * which seems to be missing the last 1 minute bar. So, this is a known bug and there seems no fix atm.
```

Comment:
Is it missing the current 1-minute bar while it is being formed? If that is the case, then it's not necessarily a bug.